chain/store_test.go

```diff
@@ -9,273 +9,37 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/ipfs/go-hamt-ipld"
-	bstore "github.com/ipfs/go-ipfs-blockstore"
-
-	"github.com/libp2p/go-libp2p-core/peer"
 
 	"github.com/filecoin-project/go-filecoin/actor"
 	"github.com/filecoin-project/go-filecoin/address"
 	"github.com/filecoin-project/go-filecoin/chain"
-	"github.com/filecoin-project/go-filecoin/consensus"
-	"github.com/filecoin-project/go-filecoin/proofs/verification"
 	"github.com/filecoin-project/go-filecoin/repo"
 	"github.com/filecoin-project/go-filecoin/state"
-	th "github.com/filecoin-project/go-filecoin/testhelpers"
 	tf "github.com/filecoin-project/go-filecoin/testhelpers/testflags"
 	"github.com/filecoin-project/go-filecoin/types"
 )
 
-// This setup stuff is a horrendous legacy holdover from when these store tests shared a
-// common monolithic setup with the syncer tests. There's no good reason for it to still exist.
-// https://github.com/filecoin-project/go-filecoin/issues/2128
-type SyncerTestParams struct {
```

Comment:
:heart: :1st_place_medal: :fireworks: YOU ROCK

Reply:
❤️ 🎧 🎸